pkg/config/sampling/sampling_test.go

```diff
@@ -80,3 +79,14 @@ func TestUpdateWithSamplingConfig(t *testing.T) {
 	assert.Len(t, options, 1)
 	assert.Equal(t, "--sampling.strategies-file=/etc/jaeger/sampling/sampling.json", options[0])
 }
+
+func TestUpdateWithSamplingConfigFileOption(t *testing.T) {
+	jaeger := v1.NewJaeger(types.NamespacedName{Name: "TestUpdateWithSamplingConfig"})
```

Comment:
nit: the name should be same as the test.

Reply:
fixed in recent commit.